src/file.js

```diff
@@ -24,22 +24,8 @@ Sk.builtin.file = function (name, mode, buffering) {
     } else if (this.name === "/dev/stderr") {
         this.fileno = 2;
     } else {
-        if (Sk.inBrowser) {  // todo:  Maybe provide a replaceable function for non-import files
```

Comment:
Now I feel like I've really missed something.  Because I thought Sk.read had to be configured to handle imports -- by checking `Sk.builtinFiles`   Did I miss some configuration change to have imports use something else?